build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -146,7 +155,11 @@
 
       <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/redist.deps.json"
                                    SectionName="runtimeTargets"
-                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(AssetsToRemoveFromDeps.Identity).exe" />
+                                   AssetPath="$(BinaryToCorehostifyRelDir)/%(RuntimeTargetsAssetsToRemoveFromDeps.Identity).exe" />
+
+      <RemoveAssetFromDepsPackages DepsFile="$(SdkOutputDirectory)/redist.deps.json"
+                                   SectionName="runtime"
+                                   AssetPath="%(RuntimeAssetsToRemoveFromDeps.Identity).dll" />
```

Comment:
The pattern here: https://github.com/dotnet/cli/pull/5415/files#diff-b03e41d91f5323926a1fd6b7df1e1d94L47 seems more consise.